doc/specs/#1571 - New Tab Menu Customization/#1571 - New Tab Menu Customization.md

```diff
@@ -129,16 +140,16 @@ nested entries for each subsequent dynamic profile generator.
     "newTabMenu": [
         { "type":"profile", "profile": "cmd" },
         { "type":"profile", "profile": "Windows PowerShell" },
-        { "type": "matchProfile", "on": "source", "match": "Microsoft.Terminal.PowerShellCore" }
+        { "type": "matchProfile", "source": "Microsoft.Terminal.PowerShellCore" }

```

Comment:
If it's a regex, I would say 
```suggestion
        { "type": "matchProfile", "source": "Microsoft\.Terminal\.PowerShellCore" }

```

Reply:
Ironically, this would still work without escaping the `.` haha. That's good though, because an unaware user should still get the expected result.

Reply:
I'VE BEEN THINKING ABOUT THAT FOR LIKE ALL OF LEAVE lol. We can totally just leave it as a regex that _just so happens to work_